shell/app/modules/msp/env-overview/service-list/pages/transaction.tsx

```diff
@@ -280,7 +288,7 @@ const Transaction = () => {
       topic,
       search,
       sort,
-      type: callType,
+      type: callType === 'undefined' ? undefined : callType,
```

Comment:
what caused a `undefined` string?

Reply:
It will be converted to a string
![image](https://user-images.githubusercontent.com/30014895/133534703-06e3d1d5-64b6-4384-9ec4-5aa646d7f863.png)

Reply:
I think it's better to move the `undefined` check to onChange function. `'undefined'` string is very strange

Reply:
The code has merged,  it will be refactor by [https://github.com/erda-project/erda-ui/pull/1247](url)